pillar.example

```diff
@@ -1,5 +1,5 @@
 elasticsearch:
-  major_version: 2
+  version: 2.4.2-1
```

Comment:
Tested with the following Pillar data:

    elasticsearch:
        version: 5.2.2-*

Results in ES 5.2.2-1 being installed.